test/unit/lib/cli/resolve-local-serverless-path.test.js

```diff
@@ -3,7 +3,7 @@
 const { expect } = require('chai');
 const path = require('path');
 const os = require('os');
-const fs = require('fs');
+const fs = require('fs').promises;
```

Comment:
cosmetic question - in some places in the code we have `const fs = require('fs').promises` and in others `const fs = require('fs')` and later in code it uses `fs.promises.something` - what is the preferred approach?

Reply:
I personally thought it's more convenient to have `fs` assigned directly to `require('fs').promises`, of course culprit is that in rare case when e.g. we want to access some non promise functions (as `createReadStream`) we will have to attach to `require('fs')` directly, and that may bite back (forgetting about that, we may write `fs.readFile`)

Maybe good alternative would be to name as `fsp` an alias to `require('fs').promises`, that should remove the confusion. What do you think?

Reply:
I like the distinction of `const fsp = require('fs').promises` to instantly know that it's just `fs.promises` module :+1:

Reply:
Good shift, updated 👍